RoboFile.php

```diff
@@ -3,6 +3,48 @@
 
 class RoboFile extends \Robo\Tasks
 {
+    public function wrap($className, $wrapperClassName = "") {
```

Comment:
Thanks for this task. Could you write in comment how to use it? So other could use it to implement their tasks too